### PR TITLE
Add Common Documentation License (CDL-1.0)

### PR DESCRIPTION
**Pull request by seabass-labrax**
Fixes issue #1109

### CODE REVIEW

**Comment by seabass-labrax** [comment]
It's getting rather late here in the UK, so unfortunately I won't be able to respond to any reviews to this pull request! I hope this helps nonetheless :)

File: src/CDL-1.0.xml [hunk 1]
```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="utf-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+  <license licenseId="CDL-1.0" isOsiApproved="false" name="Common Documentation License" listVersionAdded="3.13">
+    <crossRefs>
```
Comment (jlovejoy) [inline src/CDL-1.0.xml:4]: add "1.0" to name

**Comment by jlovejoy** [comment]
thanks @seabass-labrax !!

@swinslow  - if you haven't run the release yet, I can have a look at this later this morning

**Comment by seabass-labrax** [comment]
Thanks for reviewing this, @jlovejoy! I'd really appreciate your legal perspective on the last section - is 'EXHIBIT A' indeed an optional section as I've marked it in the XML?

File: src/CDL-1.0.xml (hunk 1 above)
Comment (seabass-labrax) [reply src/CDL-1.0.xml:4]: Thanks; will do :)

**Comment by jlovejoy** [comment]
looks good, thanks!

**Comment by swinslow** [comment]
Thank you @seabass-labrax for getting this one in on such short notice!

**Pull request merged**